presubmit.py

```diff
@@ -210,6 +210,13 @@ def lint(paths: List[Path]) -> bool:
 def pytype(paths: List[Path]) -> bool:
     """Run pytype on |path| if it is a python file. Return False if it fails
     type checking."""
+    # Check that we are on Python3.7.
```

Comment:
I think we should fail presubmit.py on anything other than python 3 (sys.version_info.major != 3).
and this should check sys.version_info.minor <= 7 and reference
https://github.com/google/pytype/issues/440 (pytype is not supported on 3.8 yet).

Reply:
>I think we should fail presubmit.py on anything other than python 3 (sys.version_info.major != 3).

I'll do it to make the code look nice but it is pointless since presubmit.py can't even start executing in Python2. It is not a syntactically valid Python2 file. Also people aren't likely to use python2 (or 4 whenever it comes out) because Make doesn't specify it.

Reply:
Done.